src/Modules/Admin/Commands.Admin/MicrosoftPowerBIMgmt.Admin.psd1

```diff
@@ -71,13 +71,13 @@ FormatsToProcess = @('Microsoft.PowerBI.Commands.Admin.format.ps1xml')
 FunctionsToExport = '*'
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = 'Add-PowerBIEncryptionKey', 'Get-PowerBIEncryptionKey', 'Get-PowerBIWorkspaceEncryptionStatus', 'Switch-PowerBIEncryptionKey', 'Set-PowerBICapacityEncryptionKey', 'Get-PowerBIActivityEvents'
+CmdletsToExport = 'Add-PowerBIEncryptionKey', 'Get-PowerBIEncryptionKey', 'Get-PowerBIWorkspaceEncryptionStatus', 'Switch-PowerBIEncryptionKey', 'Set-PowerBICapacityEncryptionKey', 'Get-PowerBIActivityEvent'
 
 # Variables to export from this module
 VariablesToExport = '*'
 
 # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
-AliasesToExport = 'Rotate-PowerBIEncryptionKey'
+AliasesToExport = 'Rotate-PowerBIEncryptionKey', 'Get-PowerBIActivityEvents'
```

Comment:
need to drop 's' here

Reply:
Backwards compat, we create an alias in case someone is using this

Reply:
ah, that makes sense. I missed the 'AliasesToExport' part.